articles/flutter-vs-kotlin/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/flutter-vs-kotlin/
+title: Flutter vs. Kotlin
+description: This article goes over a comparison of Flutter and Kotlin - talking the advantages and disadvantages of both.
+author: Bavneet Singh
+date: 2020-09-01T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/flutter-vs-kotlin/hero.jpg
+    alt: Kotlin vs Flutter image example
+---
+Let us continue from where we left off in the previous article [*Introduction to C++*](/engineering-education/intro-to-c-part1/). That article started by going over the importance of C++ as a programming language, and where it is being used today. Most importantly, we tried to go over the basic concepts of the C++ language that any developer would need to get started. If you haven't had a chance to read it already, make sure you do to get a better understanding as we move forward.
+<!--more-->
+Mobile application development has become an essential part of the software business in recent years. Previously, people used to prefer ReactNative or Flutter to develop cross-platform applications due to their low-cost features. But today, Kotlin has managed to enter into the world of cross-platform development.
+
+Since Flutter has been improved a lot in the past years, it now provides better features than ReactNative.  Kotlin has also entered the space of cross-platform development as well.
+
+In this article, we will be comparing the new languages for cross-platform development. The primary goal of this article is to help developers distinguish the best cross-development platform for them, depending on their needs and skills.
+
+### PERFORMANCE
+When it come to mobile app development, both Kotlin and Flutter are each superb with some distinctions. Kotlin can compile the code and customize it to the best needs of its user as well as the native apps.
+
+Flutter allows reloading functionality, giving the developer a choice of selection if all the different elements are performing well.
+
+Developers describe Flutter as one of the most popular cross-platform mobile framework. Kotlin, however, is described as a statistical-type programming language.
+
+### LANGUAGE
+The main advantage of Kotlin is that its programming language is quite similar to Java. Developers who have worked with Java will find it much easier to work with Kotlin. On the other hand, Flutter uses Dart from Google. Although Dart is remarkably used, it is easier to understand and operate.
```

Comment:
"Although Dart is remarkably used, it is easier to understand and operate."
If you could clarify this statement - got a bit confused - is Flutter easier to use or is Kotlin easier to use 
@bavneet25